<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>sigGenControl</class>
 <widget class="QMainWindow" name="sigGenControl">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>873</width>
    <height>814</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>sigGenControl</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTextEdit" name="history">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>200</y>
      <width>361</width>
      <height>121</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Current state history:</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="hist">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>330</y>
      <width>361</width>
      <height>121</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Commands response:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="hostEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>351</width>
      <height>22</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>HOST:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="portEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>351</width>
      <height>22</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>PORT:</string>
    </property>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>610</y>
      <width>641</width>
      <height>156</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="2" column="0">
      <widget class="QLineEdit" name="amptdEdit">
       <property name="text">
        <string/>
       </property>
       <property name="placeholderText">
        <string>Amplitude</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QRadioButton" name="radioButton">
       <property name="text">
        <string>RF OUTPUT </string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLineEdit" name="input">
       <property name="text">
        <string/>
       </property>
       <property name="placeholderText">
        <string>input command</string>
       </property>
       <property name="cursorMoveStyle">
        <enum>Qt::LogicalMoveStyle</enum>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLineEdit" name="freqEdit">
       <property name="placeholderText">
        <string>Frequency</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>500</y>
      <width>641</width>
      <height>101</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_3">
     <item row="0" column="0">
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="2" column="0">
        <widget class="QPushButton" name="sendToHostPb">
         <property name="text">
          <string>SEND COMMAND</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QPushButton" name="connPb">
         <property name="text">
          <string>CONNECT TO DEVICE</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="errPb">
       <property name="text">
        <string>CHECK ERRORS</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QCheckBox" name="checkBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>130</y>
      <width>141</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Default IP/PORT</string>
    </property>
   </widget>
   <widget class="QDial" name="dial">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>60</y>
      <width>431</width>
      <height>351</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>434</x>
      <y>20</y>
      <width>321</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>FREQUENCY :</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="amplCheckBox">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>50</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>AMPLITUDE</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>873</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
